### PR TITLE
A bunch of cosmetic changes surrounding "text" files.

### PR DESCRIPTION
**Pull request by Yamboy1**
## PR Checklist

Please check all that apply to this PR using "x":

- [x] I have checked that this PR is not a duplicate of an existing PR (open, closed or merged)
- [x] I have checked that this PR does not introduce a breaking change
- [ ] This PR introduces breaking changes and I have provided a detailed explanation below

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [x] Feature
- [ ] Code style update (formatting)
- [ ] Refactoring (no functional changes)
- [ ] Documentation changes
- [x] Other - Please describe: Style changes

## Fixes

Issue Number: N/A

## Other information

Just a few changes for the text viewer (the one which looks similar to Medium)

List of changes:
- [x] Script files are now added to the list of human readable text files
- [x] The default width has been increased, and content with longer lines will increase even more to try fit each of it's line properly.
- [x] Added a better dark theme for the code viewer, designed to integrate with the LBRY dark theme. Also removed the static scrollbar from the bottom of most code editors / viewers.
- [x] The code viewer theme now changes when the LBRY theme changes.
- [x] Add a better light theme
- [x] HTML files now have the same look and feel as the other human readable text files
- [ ] ~~Change lbry files to do the same ^~~

Other things: (figured i'd put them in the same pr rather than making two new pr's)
- [x] Add proper links to fonts in `index-electron`
- [x] Fix link rendering on the same line in markdown

### CODE REVIEW

**Comment by Yamboy1** [comment]
I'll give an example here.
Before:
image

After:
image

**Comment by Yamboy1** [comment]
There are a couple of concerns I have with these changes though. Big files, like `lbry://csv`, take ages to load and cause the app to stop responding and lbry.tv just hangs for a while.

**Comment by Yamboy1** [comment]
I think I may have messed up the history a bit, not sure what happened there, but other than that, finished the pr.

**Comment by seanyesmunt** [comment]
Thanks @Yamboy1 

I'll give this a closer look once we get the release out.

**Comment by seanyesmunt** [comment]
To fix the history/extra changes, you need to rebase against the master branch `git rebase upstream/master`

**Comment by Yamboy1** [comment]
Think i got it!

**Comment by seanyesmunt** [comment]
Great. I'll give this a closer review tomorrow.

File: ui/redux/selectors/content.js [hunk 1]
```diff
@@ -132,7 +132,7 @@ export const makeSelectIsText = (uri: string) =>
   createSelector(
     makeSelectMediaTypeForUri(uri),
     mediaType => {
-      const isText = ['text', 'document'].includes(mediaType);
+      const isText = ['document', 'script'].includes(mediaType);
```
Comment (seanyesmunt) [inline ui/redux/selectors/content.js]: There are some files that will still have the `text` `mediaType`. Just add `script` to the end without removing `text`

File: ui/scss/component/_main.scss [hunk 2]
```diff
@@ -102,7 +102,9 @@
 }
 
 .main__document-wrapper {
-  width: 40em;
+  max-width: 100%;
+  min-width: 80%;
+  width: fit-content;
```
Comment (seanyesmunt) [inline ui/scss/component/_main.scss]: I don't think this should change. If it needs to be wider for non-markdown files we can do that, but for the most part, text files should have a static width like `40em`

**Comment by Yamboy1** [comment]
If someone else wants to pick this up, i'm leaving the branch open

**Comment by Yamboy1** [comment]
Will keep working on this

File: ui/redux/selectors/content.js (hunk 1 above)
Comment (Yamboy1) [reply ui/redux/selectors/content.js]: https://github.com/lbryio/lbry-desktop/pull/3511/commits/c063ee9cfb7428314a50dd53a6b886946068aaaf

File: ui/scss/component/_main.scss (hunk 2 above)
Comment (Yamboy1) [reply ui/scss/component/_main.scss]: https://github.com/lbryio/lbry-desktop/pull/3511/commits/eeaa550a3715a96c47d638b58d269dc492893886

File: ui/component/layoutWrapperText/view.jsx [hunk 3]
```diff
@@ -20,14 +20,19 @@ type Props = {
...
-  const { uri, claim, title, nsfw } = props;
+  const { uri, claim, title, nsfw, contentType, fileType } = props;
+
+  const markdownType = ['md', 'markdown'];
+  const isMd = markdownType.includes(fileType) || contentType === 'text/markdown' || contentType === 'text/md';
 
   return (
     <div>
-      <div className="main__document-wrapper">
+      <div className={`main__document-wrapper${isMd ? '-md' : ''}`}>
```
Comment (seanyesmunt) [inline ui/component/layoutWrapperText/view.jsx]: Generally the pattern we have been following is this:
```
<div className={classNames('main__document-wrapper', { 'main__document-wrapper--markdown': isMd })}>

```

Where `main__document-wrapper` is the default style, then add any overrides to the class with `--markdown`.

Comment (Yamboy1) [reply ui/component/layoutWrapperText/view.jsx]: Thanks

**Comment by Yamboy1** [comment]
I'm a bit dissatisified by the dark theme, so I might make a few tweaks there before this gets merged

**Comment by seanyesmunt** [comment]
@Yamboy1 Should I review this or are you still planning to do more work with the dark theme? Happy to review or wait.

**Comment by Yamboy1** [comment]
Sorry, I've been real busy with school and stuff recently. I'll get onto it

**Comment by Yamboy1** [comment]
Should be good to merge now :smile:

**Comment by Yamboy1** [comment]
wait

**Comment by Yamboy1** [comment]
Seems good

**Pull request merged**